guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,25 +1,17 @@
-[id="Upgrading_the_External_Database_{context}"]
+[id="Upgrading_the_External_Database{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+If your {Project} deployment uses an external database, the database will be upgraded when running the {foreman-installer} on the {Project} server.
```

Comment:
```suggestion
If your {Project} deployment uses an external database, the database will be upgraded when running {foreman-installer} on the {Project} server.
```
or
```suggestion
If your {Project} deployment uses an external database, the database will be upgraded when running the {foreman-installer} utility on the {Project} server.
```
At least I think that the convention is not to use a definite article with the installer like this, someone please correct me if I'm wrong.

Reply:
I'll go with the first suggestion :+1: